velocity/src/main/java/io/github/retrooper/packetevents/manager/InternalVelocityPacketListener.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of packetevents - https://github.com/retrooper/packetevents
+ * Copyright (C) 2022 retrooper and contributors
```

Comment:
When either creating new files or updating old ones, the copyright year should be changed to the current year.

Reply:
Yep, changed it.